packages/particle-network/src/index.ts

```diff
@@ -0,0 +1,110 @@
+import {
+	WalletInit,
+	EIP1193Provider,
+	ProviderRpcError,
+	ProviderRpcErrorCode,
+	ProviderAccounts,
+} from '@web3-onboard/common';
+import { createEIP1193Provider } from '@web3-onboard/common';
+import { ParticleNetwork, Config } from '@particle-network/auth';
+import { ParticleProvider } from '@particle-network/provider';
```

Comment:
We try to dynamically import any dependencies (that are type imports) to keep weight off the client until selected by a user. Can we update these to be imported dynamically?

Reply:
I can do this with a few objects, such as `createEIP1193Provider`, `ParticleNetwork`, and `ParticleProvider` but run into issues when trying to convert various type imports; I've shifted the import structure here to be similar to the Web3Auth package, let me know if this works.